sql/load_modifications.sql

```diff
@@ -26,4 +27,15 @@ CREATE TABLE reviewed_modified_names(
     reviewed text
 );
 
-\COPY reviewed_modified_names FROM 'data/ipis_modified_names.csv' DELIMITER ',' CSV HEADER;
\ No newline at end of file
+\COPY reviewed_modified_names FROM 'data/ipis_modified_names.csv' DELIMITER ',' CSV HEADER;
+
+
+DROP TABLE IF EXISTS modifications_by_ID;
```

Comment:
Using values the `field` column to partition the modifications table is not sustainable.  What happens if a new value is added to the `field` column and the match should be done by uid?  An idea is to modify and clean up the values in the modifications table so that records that should be matched by value do not have a uid value in the uid column.  This is more similar to how the PLUTO modifications table is structured.